apps/meteor/app/livechat/server/api/lib/livechat.ts

```diff
@@ -164,6 +164,8 @@ export async function settings({ businessUnit = '' }: { businessUnit?: string }
 			limitTextLength:
 				initSettings.Livechat_enable_message_character_limit &&
 				(initSettings.Livechat_message_character_limit || initSettings.Message_MaxAllowedSize),
+			livechatLogo: initSettings.Assets_livechat_widget_logo,
+			hideWatermark: initSettings.Livechat_hide_watermark,
```

Comment:
I'm missing some validation somewhere about the license.

Reply:
Agree, also, when the env is fully CE, then these settings will be `null`. Widget should handle those cases as well.

Reply:
Issue addressed on #31970